### PR TITLE
Core: fix deprecation warning in for utcnow()

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding?
`DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
  self.buildtime = datetime.datetime.utcnow()`

## How was this tested?
running setup.py build_exe on py 3.12

### CODE REVIEW

**Review by silasary — APPROVED** [review]

**Review by nicholassaylor — APPROVED** [review]
LGTM, this should work now that 3.8 is binned

File: setup.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def run(self) -> None:
                                   f"{ex}\nPlease close all AP instances and delete manually.")
 
         # regular cx build
-        self.buildtime = datetime.datetime.utcnow()
+        self.buildtime = datetime.datetime.now(datetime.UTC)
```
Comment (nicholassaylor) [inline setup.py:324]:
```suggestion
        self.buildtime = datetime.datetime.now(datetime.timezone.UTC)
```
`datetime.UTC` was added in 3.11, this should be 3.10 compatible

**Review by nicholassaylor — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -321,7 +321,7 @@ def run(self) -> None:
                                   f"{ex}\nPlease close all AP instances and delete manually.")
 
         # regular cx build
-        self.buildtime = datetime.datetime.utcnow()
+        self.buildtime = datetime.datetime.now(datetime.timezone.UTC)
```
Comment (nicholassaylor) [inline setup.py:324]:
```suggestion
        self.buildtime = datetime.datetime.now(datetime.timezone.utc)
```
Capitalization is hard

**Review by black-sliver — APPROVED** [review]
```python
>>> from datetime import datetime, timezone
>>> datetime.utcnow().hour == datetime.now(timezone.utc).hour
True
```

**Review by nicholassaylor — APPROVED** [review]

**Pull request merged**